src/pybind/mgr/cephadm/schedule.py

```diff
@@ -304,7 +304,7 @@ def get_candidates(self) -> List[DaemonPlacement]:
                     ls.append(h)
                 else:
                     logger.info(
-                        f"Filtered out host {h.hostname}: could not verify host allowed virtual ips")
+                        f"Filtered out host {h.hostname}: host does not belong to mon public_network")
```

Comment:
we're trying to invent a message here, but only `self.filter_new_host` can provide a proper message. can we move this this log message to the implementation of self.filter_new_host ?